python/lsst/cp/pipe/linearity.py

```diff
@@ -265,7 +270,28 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         # Use the dimensions to set calib/provenance information.
         inputs['inputDims'] = dict(inputRefs.inputPtc.dataId.required)
 
+        # Add calibration provenance info to header.
+        if self.config.doHeaderProvenance:
+
+            kwargs = dict()
+            reference = getattr(inputRefs, "inputPtc", None)
+            self.log.info(str(reference.run) + " and " + str(reference.id))
+            if reference is not None and hasattr(reference, "run"):
+                runKey = "PTCRUN"
```

Comment:
It's annoying that astropy can't handle the long names.  Can these start with `PTC_` so they're somewhat easier to read?  They'll stay below 8 characters even with that.